### DISCUSSION TITLE
Unable to change theme color of chosed game in library

### DISCUSSION THREAD

**Discussion by Portnoon** [SleepDaemon/Modern-Steam#5] [Q&A]
I tried to customize the theme color through custom.styles yet it only changes the color of the mainmenu on top, when I chose a game in my library, it still appear as the default purple.
So, how do I actually customize the theme color?
Any idea plz
![1](https://user-images.githubusercontent.com/124462295/216784547-bd43d6bf-bbbb-49c2-99b5-ea4818f89465.png)
?

**Comment by SleepDaemon** [comment]
> I tried to customize the theme color through custom.styles yet it only changes the color of the mainmenu on top, when I chose a game in my library, it still appear as the default purple. So, how do I actually customize the theme color? Any idea plz ![1](https://user-images.githubusercontent.com/124462295/216784547-bd43d6bf-bbbb-49c2-99b5-ea4818f89465.png) ?

Hi, I pretty sure you didn't edit the correct file. But I am not sure either cause I never bother changing the color. I'll look into this later but for now I will link this discussion to a issue.
